Source/WebKit/UIProcess/Launcher/cocoa/ExtensionProcess.mm

```diff
@@ -71,7 +79,32 @@
     RetainPtr<BEProcessCapabilityGrant> grant;
     WTF::switchOn(m_process, [&] (auto& process) {
         grant = [process grantCapability:capability error:&error];
+    }, [&] (RetainPtr<_SEExtensionProcess> process) {
```

Comment:
Doesn't need to capture `&`. Also, does the compiler require us to specify `process`?